src/components/Layout/Header.tsx

```diff
@@ -92,6 +205,9 @@ const Header: FunctionComponent<Props> = props => {
                                     className="btn btn-simple px-2 py-2 font-weight-bolder"
                                     href="https://sourcegraph.com/search"
                                     title="Search code"
+                                    data-button-style={buttonStyle.text}
+                                    data-button-location={buttonLocation.nav}
+                                    data-button-type="cta"
```

Comment:
@bretthayes Re: Button Tracking. I noticed we are missing attributes on the `right-nav` buttons. Are we able to add them on `Nav.Link`? I initially added them on an `a` wrapper within `Nav.Link`, but this led styling to load unpredictably. Please let me know what you think--I'm happy to iron this out!

Reply:
Yes we should be able to add tracking/data attributes to `Nav.Link`